Server/src/controllers/dataUploadController.ts

```diff
@@ -9,20 +11,42 @@ import { DataUploadService } from '../services/DataUploadService'
  */
 
 export class DataUploadController {
-    private dataService: DataUploadService
+    private dataService: any
+    private dataSet: IDataSetModel
 
     constructor() {
     }
 
     async createRequest(request: Request, response: Response): Promise<Response> {
         if (!request.body) {
             response.status(400).json({
-                message: "No Json to Upload Received"
+                message: "No Data to Upload"
             })
         }
         else {
-            this.dataService = new DataUploadService(request.body);
             try {
+                this.dataSet = request.body
+                this.dataService = new UnapprovedUploadService(this.dataSet)
```

Comment:
dataService shouldnt be any but some interface that UnapprovedUploadService implements and other services that are used in the dataUploadController

Reply:
Yeah I forgot to get rid of the any

Reply:
Fixed in pending commit